code/game/machinery/status_display.dm

```diff
@@ -32,6 +36,55 @@
 	var/index1 // display index for scrolling messages or 0 if non-scrolling
 	var/index2
 
+/obj/item/wallframe/status_display
+	name = "status display frame"
+	desc = "Used to build status displays, just secure to the wall."
+	icon_state = "unanchoredstatusdisplay"
+	custom_materials = list(/datum/material/iron=14000, /datum/material/glass=8000)
+	result_path = /obj/machinery/status_display
+
+//Copy pasted decon paths from newscaster.dm
+/obj/machinery/status_display/attackby(obj/item/I, mob/living/user, params)
+	if(I.tool_behaviour == TOOL_WRENCH)
```

Comment:
Use `tool_act` instead for any construction, `tool_act_secondary` for any deconstruction.